.gitignore

```diff
@@ -185,7 +185,6 @@ dmypy.json
 
 # Protos
 sdk/python/docs/html
-sdk/python/feast/protos/
 sdk/go/protos/
```

Comment:
Can probably removes all of the protos from the git ignore, yeah?

Reply:
I think those are golang ones, I'll look into it later, don't want to push something extra inadvertently.